src/aggregated_bls/party_i.rs

```diff
@@ -52,11 +53,45 @@ impl Keys {
 
     pub fn combine_local_signatures(sigs: &[SIG]) -> BLSSignature {
         let (head, tail) = sigs.split_at(1);
-        let sig_sum = tail.iter().fold(head[0], |x, acc| x + acc);
+        let sig_sum = tail.iter().fold(head[0], |acc, x| acc + x);
         BLSSignature { sigma: sig_sum }
     }
 
     pub fn verify(signature: &BLSSignature, message: &[u8], apk: &APK) -> bool {
```

Comment:
Similar to `core_aggregate_verify` do we need to check if the public key is a valid element of the subgroup and not the identity here?